index.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+/* eslint-disable import/no-cycle */
+import addToList from './modules/add.js';
+import Books from './modules/books.js';
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.date = Book.timeStamp();
+    this.id = Math.random();
+  }
+
+  static timeStamp() {
+    const date = new Date();
+    return date.getMinutes();
+  }
```

Comment:
It will be better to make this function an `es6 syntax` function. Remember we need to use `es6` for this milestone. 

**HINT**
```
const timeStamp = () => { your code }
```